transform/voxel_around_line.go

```diff
@@ -17,7 +17,7 @@ import (
 	geodesy "github.com/trajectoryjp/geodesy_go/coordinates"
 )
 
-// GetSpatialIdsWithinRadiusOfLine
+// GetExtendedSpatialIdsWithinRadiusOfLine
 // 直線と線からradiusの距離以内の拡張空間IDを取得する
 //
 // 引数：
```

Comment:
Append the comment of skipsMeasurement.

Reply:
resolved here https://github.com/trajectoryjp/spatial_id_go/pull/19/commits/9431a3af06191bc384802142950fd0d3eae8b174